packages/experimental/Switch/src/Switch.tsx

```diff
@@ -1,11 +1,37 @@
 /** @jsx withSlots */
```

Comment:
is there any change in this file code from *.win32 variant?

Reply:
Ok sure , just a thought will iOS will create their specific file ? 
[mapPipelineToTheme.ts](https://github.com/microsoft/fluentui-react-native/pull/2372/files/ed1538ec77911880df86b3e95208e9f137b8080c#diff-9b5eeaa8397d44a2c9ceaefd591fe2936905759e53fb06dfa70bc1c014c2348c)

Reply:
No , nothing has changed from win32 , just the exact copy of it.